hardhat.config.ts

```diff
@@ -0,0 +1,35 @@
+import "@nomiclabs/hardhat-ethers";
+import "@nomiclabs/hardhat-web3";
+import "@nomiclabs/hardhat-waffle";
+import "babel-register";
+import "babel-polyfill";
+import "@typechain/hardhat";
+import "hardhat-gas-reporter";
+import "solidity-coverage";
+import "@nomiclabs/hardhat-etherscan";
+
+/**
+ * @type import('hardhat/config').HardhatUserConfig
+ */
```

Comment:
We'll need this if we are going to add typings 
```suggestion
import { HardhatUserConfig } from "hardhat/types";

```